contents/handbook/growth/sales/billing.md

```diff
@@ -28,9 +28,10 @@ We use metadata on these prices to allow the Billing Service to appropriately lo
 * `posthog_product_key`: `posthog_analytics` | `session_replay` | ... -> This allows PostHog to find and map the relevant products. **Important:** There should never be more than 1 Stripe product with the same `posthog_product_key`. The list of keys is defined in the main billing config.
 
 **On Stripe Product Prices**
-The following keys are used to manage Startup prices, might be deprecated:
+The following keys are used to manage Startup prices:
+* `plan` - Any Startup plan prices must have the `plan` metadata set to `startup` to have their subscription automatically moved to the default (paid) prices. If not, when their subscription ends they will instead be switched to the free plans for all products. 
 * `valid_days` -> The number of days a price is valid for, before automatically switching to another plan (the `default` plan unless `move_to_price_id` is set). Useful to create pricing that is only valid for a specific period, e.g. for the startup plans. Note: if more than one price with `valid_days` is added to a subscription, the validity period will be the *shortest* of the two, before resetting all plans to the default ones 
```

Comment:
@raquelmsmith comment/question on the valid days - we might want to put people onto the startup plan for either 6 or 12 months depending on which YC batch they're in.  Will this work or would we need a separate price for each?

Reply:
I think you'll need a separate price - one for 6-month plans with `valid_days` set to 180 and another with valid_days set to 365.

Reply:
To be frank, I haven't looked into the `valid_days` thing so I'm not sure how it works, but I feel quite certain that we can't have two metadata properties that are the same but with different values on the same price :)